CoreModels/QAActivity.cs

```diff
@@ -16,9 +16,15 @@ public class QAActivity
 
         public DateTime DateCreated { get; set; }
 
+        public bool IsDeleted { get; set; }
+
+        public DateTime? DateDeleted { get; set; }
+
         public QAActivity()
         {
             this.DateCreated = DateTime.Now;
+            this.IsDeleted = false;
```

Comment:
no need to assign IsDeleted and Date deleted here. Give default value for isDeleted in table and keep date deleted as nullable column